diff --git a/libcxx/CMakeLists.txt b/libcxx/CMakeLists.txt
index ec1869ddc..d64def6c2 100644
--- a/libcxx/CMakeLists.txt
+++ b/libcxx/CMakeLists.txt
@@ -449,7 +449,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${LIBCXX_LIBRARY_DIR})
 #                             These libraries are exposed in the linker script.
 set(LIBCXX_COMPILE_FLAGS "")
 set(LIBCXX_LINK_FLAGS "")
-set(LIBCXX_LIBRARIES "")
+set(LIBCXX_LIBRARIES "-lgcc")
 set(LIBCXX_INTERFACE_LIBRARIES "")

 # Include macros for adding and removing libc++ flags.
